src/System.Runtime.Extensions/src/System/Collections/Hashtable.cs

```diff
@@ -953,7 +953,7 @@ private void Insert(Object key, Object nvalue, bool add)
                 {
                     if (add)
                     {
-                        throw new ArgumentException(SR.Format(SR.Argument_AddingDuplicate__, _buckets[bucketNumber].key, key));
+                        throw new ArgumentException(SR.Format(SR.Argument_AddingDuplicate, key));
```

Comment:
@danmosemsft related to https://github.com/dotnet/corefx/pull/29280/files#r183536136 i'm not convinced, the message was more detailed.

Reply:
You're quite right, I was wrong. This change loses information. Please keep `Argument_AddingDuplicate__`, but perhaps rename like `Argument_AddingDuplicate_OldAndNewKeys` or something like that.